client/src/component/HomePage.jsx

```diff
@@ -0,0 +1,28 @@
+import React, { useState, useEffect } from "react";
+import Video from "./component/Video";
+import Header from "./component/Header";
+import AddVideo from "./component/AddVideo";
+
+function HomePage() {
+  const [videos, setVideos] = useState([]);
+
+  const getData = async () => {
+    const res = await fetch("http://localhost:4000/api/videos");
+    const data = await res.json();
+    setVideos(data);
+  };
+
+  useEffect(() => {
+    getData();
+  }, []);
+
+  return (
+    <div>
+      <Header />
+      <AddVideo videoData={videos} />
+      <Video videoData={videos} />
```

Comment:
If I was to be super pedantic, you could replace the <div> with <>. This will lower the bundle size and DOM elements, making our website lighter and faster :)